torch/onnx/_internal/fx/fx_exporter.py

```diff
@@ -2,6 +2,7 @@
 from typing import Any, Callable, Mapping, Sequence
 
 import torch._ops
+import torch.fx
```

Comment:
for intellisense/pylance not complaining `"fx" is not a known member of module "torch"` for `torch.fx.GraphModule`.